content/200-concepts/100-components/database-drivers.mdx

```diff
@@ -17,15 +17,21 @@ Prisma Client can connect and run queries against your database using JavaScript
 
 Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
 
-![Query flow from the user application to the database using Prisma Client and driver adapters](./images//drivers/qe-query-engine-adapter.png)
+![Query flow from the user application to the database using Prisma Client and driver adapters](./images/drivers/qe-query-engine-adapter.png)
 
```

Comment:
Here would be a good spot to introduce that there are different kind of driver adapters maybe.